test/tests/problems/eigen_problem/ipm.i

```diff
@@ -79,6 +79,7 @@
 
 [Executioner]
   type = EigenExecutioner
+  eigen_solve_type = KRYLOVSCHUR
```

Comment:
Since eigen executioner is separated from Steady, we can just use `solve_type` here.